lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -116,9 +116,10 @@
                                 title: gettext('Time Zone'),
                                 valueAttribute: 'time_zone',
                                 helpMessage: gettext(
-                                    'Select the time zone for displaying course dates. If you do not specify a ' +
-                                    'time zone here, course dates, including assignment deadlines, are displayed in ' +
-                                    'Coordinated Universal Time (UTC).'
+                                    'Select the time zone for displaying course dates. ' +
```

Comment:
This looks to be a little thorny -- all of the dropdowns are rendered as a part of a template, and we don't have a straightforward way (that I can find) to render text in the 'Null' field. https://openedx.atlassian.net/browse/TNL-5839